code/modules/shuttle/emergency.dm

```diff
@@ -585,11 +588,37 @@
 				var/destination_dock = "emergency_away"
 				if(is_hijacked() || elimination_hijack())
 					// just double check
-					SSmapping.lazy_load_template(LAZY_TEMPLATE_KEY_NUKIEBASE)
-					destination_dock = "emergency_syndicate"
-					minor_announce("Corruption detected in \
-						shuttle navigation protocols. Please contact your \
-						supervisor.", "SYSTEM ERROR:", sound_override = 'sound/misc/announce_syndi.ogg')
+					if(hijacker.mind)
+
+						var/destination = hijacker.mind.get_desired_hijack_destination()
+
+					// To add another destination, go to the _DEFINES\antagonists to name your desitnation and give it a number to define it.
+					// Remember to add it to your antagonist's hijack_location varible as well.
+					// Then define where you want the shuttle to dock by by choosing or creating a new docking port in switch statement
+					// get_desired_hijack_destination() takes only the destination with the highest number in the case of an mind with multiple antagonist datums
+						switch(destination)
```

Comment:
Add a default branch here that calls `stack_trace` with the antagonist type and its destination so we can more easily find bad ones.

Reply:
Let me know if the verbiage isn't to your liking.